app/controllers/jobs_controller.rb

```diff
@@ -1,12 +1,21 @@
+require 'page_setup'
+
 class JobsController < ApplicationController
   def index
     @company = Company.find(params[:company_id])
-    @jobs = @company.jobs
+    if params[:sort]
```

Comment:
this chunk is doing a lot of a controller - could we refactor this into maybe a PORO or something else?